src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -106,9 +148,29 @@ public enum NNFWType
         /// </summary>
         Tensorflow,
         /// <summary>
-        /// Neural Network Inference framework, which is developed by SR
+        /// Neural Network Inference framework, which is developed by SR (Samsung Research).
         /// </summary>
         NNFW,
+        /// <summary>
+        /// Intel Movidius Neural Compute SDK (libmvnc). (Since 8)
```

Comment:
I suggest using explicite tag instead of a comment in all enum values.
```suggestion
        /// Intel Movidius Neural Compute SDK (libmvnc).
        /// <since_tizen> 8 </since_tizen>
```

Reply:
Updated